app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -54,7 +54,19 @@ def hash_for_pdf
         "SpousesCU Partners SSN if filing jointly": get_address, # address text field
         "CountyMunicipality Code See Table page 50": @xml_document.at("ReturnHeaderState Filer USAddress CityNm")&.text,  # city / town text field
         "State": @xml_document.at("ReturnHeaderState Filer USAddress StateAbbreviationCd")&.text,
-        "ZIP Code": @xml_document.at("ReturnHeaderState Filer USAddress ZIPCd")&.text
+        "ZIP Code": @xml_document.at("ReturnHeaderState Filer USAddress ZIPCd")&.text,
+
+        # line 6 exemptions
+        "Check Box39": @xml_document.at("Exemptions SpouseCuRegular")&.text == "X" ? "Yes" : "Off",
+        "Check Box40": "Off",
+        "Domestic": get_line_6_exemption_count,
+        "x  1000": get_line_6_exemption_count * 1000,
+
+        # line 7 exemptions
+        "Check Box41": @xml_document.at("Exemptions YouOver65")&.text == "X" ? "Yes" : "Off",
+        "Check Box42": @xml_document.at("Exemptions SpouseCuPartner65OrOver")&.text == "X" ? "Yes" : "Off",
```

Comment:
[pebble] Didn't realize the PDF value for a checkbox was "Yes" or "Off" hahaha. Can we extract this into a private utility function for getting the PDF checkbox value so that we're not repeating this ternary in many places?

Reply:
Yeah "Yes" and "Off" is not my favorite either. Refactoring...